spec/game_spec.rb

```diff
@@ -189,31 +181,32 @@ def print_message(string)
   end
 
   it 'ends the game after the board is full with no winning combinations' do
-    @console.stub_get_input(['1', '2', '3', '4', '5', '8', '6', '9', '7'])
+    @console.stub_get_input(['1', '2', '3', '4', '5', '7', '6', '9', '8'])
```

Comment:
I'm curious about this change - was it necessary to get the test to pass for some reason?

Reply:
I got worried when I saw this test fail and then I realized I wrote the test incorrectly the first time and it was passing because it was returning board.full? as true and now that it is checking for a draw it failed because I had a winning combination in there by mistake. 

original method
`  def self.game_over?(board)
    winning_combination?(board) || board.full?
  end`